plans/switch_master.pp

```diff
@@ -0,0 +1,47 @@
+plan peadm::switch_master (
+  TargetSpec $nodes,
+  String $master,
+  Boolean $sign_csr = false,
+
+){
+
+  # Extract the target node name and convert to certname array
+   $certname = get_targets($nodes).map |$a| { $a.name }
+
+  # Task to stop Puppet and change the Puppet Server name in puppet.conf
+    run_task ('peadm::agent_switchover',$nodes,
```

Comment:
Style check: no space between function name and opening parenthesis.

Should probably do a `puppet-lint` pass on this file; I think there are a few other style fixes to be made too. From a quick point-in-time check:

```
% puppet-lint switch_master.pp
ERROR: two-space soft tabs not used on line 9
ERROR: trailing whitespace found on line 24
ERROR: trailing whitespace found on line 44
WARNING: double quoted string containing no variables on line 46
```

Reply:
updated puppet-lint style issues